### PR TITLE
Updated NTP bootstrap setup function for 5.0

### PR DESCRIPTION
**Pull request by maka-rubrik**
Updated NTP bootstrap setup function for 5.0.

# Description

Expected Behavior
NTP Configuration to be successfully applied during the bootstrap of a 5.0 Cluster

Current Behavior
The NTP Configuration does not apply.Failure Information (for bugs)

Pre 5.0 "/internal/cluster/{id}/bootstrap" API config  
  "ntpServers": [
    "string"
  ],

Post 5.0 "/internal/cluster/{id}/bootstrap" API config  
"ntpServerConfigs": [
    {
      "server": "string",
      "symmetricKey": {
        "keyId": 0,
        "key": "string",
        "keyType": "string"
      }
    }
  ],

Added new flag to check if bootstrap will be on pre-5.0 or post-5.0 cdm version. The correct ntp configuration schema will be applied. 

## Related Issue

This project only accepts pull requests related to open issues.

* If suggesting a new feature or change, please discuss it in an issue first.
* If fixing a bug, there should be an issue describing it with steps to reproduce

_Please link to the issue here_

## Motivation and Context

Why is this change required? What problem does it solve?

## How Has This Been Tested?

* Please describe in detail how you tested your changes.
* Include details of your testing environment, and the tests you ran to see how your change affects other areas of the code, etc.

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

Go over all the following points, and put an `x` in all the boxes that apply. If you're unsure about any of these, don't hesitate to ask. We're here to help!
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **[CONTRIBUTION](https://github.com/rubrikinc/rubrik-sdk-for-python/blob/master/CONTRIBUTING.md)** document.
- [ ] I have updated the CHANGELOG file accordingly for the version that this merge modifies.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by drew-russell** [comment]
Feedback left via Slack.

**Comment by drew-russell** [comment]
@maka-rubrik any movement here?

**Comment by maka-rubrik** [comment]
@drew-russell Just pushed out a new version using a pre-existing version check that was part of the SDK.

File: docs/setup_cluster.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
 
 Issues a bootstrap request to a specified Rubrik cluster
 ```py
-def setup_cluster(cluster_name, admin_email, admin_password, management_gateway, management_subnet_mask, node_config=None, enable_encryption=True, dns_search_domains=None, dns_nameservers=None, ntp_servers=None, wait_for_completion=True, timeout=30)
+def setup_cluster(cluster_name, admin_email, admin_password, management_gateway, management_subnet_mask, node_config=None, enable_encryption=True, dns_search_domains=None, dns_nameservers=None, ntp_servers=None, pre_cdm5=False, wait_for_completion=True, timeout=30)
```
Comment (drew-russell) [inline docs/setup_cluster.md]:
```python
def setup_cluster(cluster_name, admin_email, admin_password, management_gateway, management_subnet_mask, node_config=None, enable_encryption=True, dns_search_domains=None, dns_nameservers=None, ntp_servers=None, pre_cdm5=False, wait_for_completion=True, timeout=30)
```

We should be able to remove the `pre_cdm5` variable now, correct?

File: docs/setup_cluster.md [hunk 2]
```diff
@@ -50,6 +50,7 @@ mDNS name resolution can be verified on systemd-based Linux systems using the co
 | dns_search_domains  | str  | The search domain that the DNS Service will use to resolve hostnames that are not fully qualified.  |         |    None     |
 | dns_nameservers  | list  | IPv4 addresses of DNS servers.  |         |    [8.8.8.8]     |
 | ntp_servers  | list  | FQDN or IPv4 address of a network time protocol (NTP) server.  |         |    [pool.ntp.org]     |
+| pre_cdm5  | bool  | Flag to determine if bootstrap will be on a pre-cdm 5.0 version. |         |    False     |
```
Comment (drew-russell) [inline docs/setup_cluster.md]: Similar to above ... we should be able to remove the `pre_cdm5` documentation here.

File: rubrik_cdm/rubrik_cdm.py [hunk 3]
```diff
@@ -358,7 +358,15 @@ def setup_cluster(self, cluster_name, admin_email, admin_password, management_ga
...
         bootstrap_config["dnsNameservers"] = dns_nameservers
         bootstrap_config["dnsSearchDomains"] = dns_search_domains
-        bootstrap_config["ntpServers"] = ntp_servers
+
+        if self.minimum_installed_cdm_version(5.0) is False:
+            bootstrap_config["ntpServers"] = ntp_servers
+        else:
+            bootstrap_config["ntpServerConfigs"] = []
+            for ntp in ntp_servers:
+                tempNtp = {}
```
Comment (drew-russell) [inline rubrik_cdm/rubrik_cdm.py]: `tempNtp = {}` should be `temp_ntp` in terms of formatting. 

Can we also come up with something more descriptive for the variable name? Ideally someone who has never looked at the code understands what is happening here right off the bat.

Also, we're you able to verify this call works on a non-bootstrapped cluster?

**Comment by drew-russell** [comment]
@maka-rubrik any updates on the requested changes above?

File: rubrik_cdm/rubrik_cdm.py (hunk 3 above)
Comment (maka-rubrik) [reply rubrik_cdm/rubrik_cdm.py]: Will do. I'll be able to complete a verification test by next week and submit the changes requested.

Comment (maka-rubrik) [reply rubrik_cdm/rubrik_cdm.py]: @drew-russell Made the changes you requested and tested them.

**Pull request merged**